packages/front-end/pages/vercel.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+import { getApiHost } from "@/services/env";
+import { useProject } from "@/services/DefinitionsContext";
+
+const VercelPage = () => {
```

Comment:
Just a nit, but do we want to nest this page inside the `oauth` directory?  We do the same for other auth callback pages we have.  Just thinking we should leave the top-level `/vercel` route open in case we want to use it for something else in the future.